package.json

```diff
@@ -40,7 +40,9 @@
   "license": "MIT",
   "readmeFilename": "README.md",
   "gitHead": "67eb8fe3c26205efc94ae95b3ef37a469eef0b81",
-  "dependencies": {},
+  "dependencies": {
+    "grunt-cli": "^1.3.2"
```

Comment:
Bootstrap-slider has no `npm` dependencies. Please rebase your local branch and ignore/discard this change.

If you're having trouble with write access, then you need administrator access.

Try running this, to install `grunt-cli` globally

```bash
sudo npm install grunt-cli -g
```

And just run `grunt-dev` to create a local development server for testing Bootstrap-Slider.